boxes/Dockerfile

```diff
@@ -14,4 +14,5 @@ WORKDIR /usr/src/boxes
 ENV AZTEC_NARGO=/usr/src/noir/target/release/nargo
 ENV AZTEC_CLI=/usr/src/yarn-project/cli/aztec-cli-dest
 RUN yarn && yarn build
+RUN npx -y playwright@1.41.1 install --with-deps
```

Comment:
When building dockerfiles, things that change rarely should be higher up the file so that you don't have to rerun the command every time when building. This should probably go after package installation.

Reply:
Oh indeed it will cache previous steps, didn’t notice it. I’ll move it up!